eng/scripts/Test-Code.ps1

```diff
@@ -259,14 +259,16 @@ try {
     $resultsArg = "--results-directory '$TestResultsPath'"
     $loggerArg = "--logger 'trx' --logger 'console;verbosity=detailed'"
 
-    $command = "dotnet test $coverageArg $resultsArg $loggerArg"
+    $buildCommand = "dotnet build --verbosity:quiet"
+    $testCommand = "dotnet test --no-build --no-restore $coverageArg $resultsArg $loggerArg"
 
     if($Members.Count -gt 0) {
         $memberFilterString = $Members | ForEach-Object { "FullyQualifiedName~$_" } | Join-String -Separator '|'
-        $command += " --filter '$memberFilterString'"
+        $testCommand += " --filter '$memberFilterString'"
     }
 
-    Invoke-LoggedCommand -Command $command -AllowedExitCodes @(0, 1)
+    Invoke-LoggedCommand $buildCommand
```

Comment:
The reason the command allows 0/1 I believe is to allow all the test to run instead of failing fast. Is that correct @hallipr ?

Reply:
All the tests should run as part of the `dotnet test` invocation, `AllowedExitCodes` are only check after the expression finishes running